Sources/EntitiesMP/Fish.es

```diff
@@ -283,7 +283,7 @@ procedures:
     // declare yourself as a model
     InitAsModel();
     // fish must not go upstairs, or it will get out of water
-    SetPhysicsFlags((EPF_MODEL_WALKING|EPF_HASGILLS)&~EPF_ONBLOCK_CLIMBORSLIDE|EPF_ONBLOCK_SLIDE);
+    SetPhysicsFlags(((EPF_MODEL_WALKING|EPF_HASGILLS)&~EPF_ONBLOCK_CLIMBORSLIDE)|EPF_ONBLOCK_SLIDE);
```

Comment:
it's not necessarily easy to notice - I think this specifies what happens when the fish collides with a block, which probably doesn't happen all that often (not in water and I guess it's hard - but maybe not impossible - to get the fish out of water?).
The original code (and the way you set the parenthesis) it would slide along the (steep?) block. Out of the water that'd make sense, inside the water not so much? (No idea if inside the water EPF_ONBLOCK_SLIDE is even checked)